packages/runtime/src/runtimes/react/index.tsx

```diff
@@ -62,11 +67,16 @@ export function createReactRuntime(store: ReactPage.Store): ReactRuntime {
         unregisterReactComponent()
       }
     },
+    getPropControllerDescriptors() {
```

Comment:
Is there a way we can avoid exposing this in the public `ReactRuntime` API?

Reply:
Yea I thought so too. I moved it to a new runtime called `InternalReactRuntime`.

Reply:
We already have internal APIs that we can access directly by using the Redux store and selectors. The runtime API was created specifically to wrap the internal API and expose a public API. So having an "internal" runtime API is a bit roundabout.

Reply:
We need to access the store, which is the `contextDefaultValue`. Should we just access this variable directly?

Otherwise how could we access the store from `getServerSideProps`?